prisma/schema.prisma

```diff
@@ -25,6 +25,23 @@ model User {
     gift      Gift[]
     Session   Session[]
     Feedback  Feedback[]
+    Person    Person[]
```

Comment:
Tämä pitäisi olla ok, koska yhdellä käyttäjällä voi olla monta ihmistä lisättynä (one-to-many)